apps/take_picture/app.c

```diff
@@ -0,0 +1,197 @@
+/**
+ * This example takes a picture every 5s.
+   Example src: https://github.com/espressif/esp32-camera.git 
```

Comment:
@ralph-lange parts from this example come from the example [here](https://github.com/espressif/esp32-camera/blob/master/examples/take_picture.c). Do we need to mention that in 3rd-party-licenses.txt?
The license there is also Apache 2.0

Reply:
Yes, all details (version, copyright statement, original license text) have to be provided in the `3rd-party-licenses.txt` file, please. Furthermore, a short summary (one list item with one or two sentences) should go into the _License_ section in `README.md`.